content/en/docs/release-notes/rn-armory-spinnaker/armoryspinnaker_v2-26-0.md

```diff
@@ -0,0 +1,316 @@
+version: 02.26.00
+description: >
+  Release notes for Armory Enterprise v2.26.0
+---
+
+## 2021/05/20 Release Notes
+
+> Note: If you're experiencing production issues after upgrading Spinnaker, rollback to a [previous working version]({{< ref "upgrade-spinnaker#rolling-back-an-upgrade" >}}) and please report issues to [http://go.armory.io/support](http://go.armory.io/support).
+
+## Required Halyard or Operator version
+
+To install, upgrade, or configure Armory 2.26.0, use one of the following tools:
+
+- Armory-extended Halyard 1.12 or later
+- Armory Operator 1.2.6 or later
+
+## Security
+
+Armory scans the codebase as we develop and release software. Contact your Armory account representative for information about CVE scans for this release.
+
+## Breaking changes
+<!-- Copy/paste from the previous version if there are recent ones. We can drop breaking changes after 3 minor versions. Add new ones from OSS and Armory. -->
+
+#### Suffixes for the Kubernetes Run Job stage
+
+The `kubernetes.jobs.append-suffix` parameter no longer works. The removal of this parameter was previously announced as part of a [breaking change](https://docs.armory.io/docs/release-notes/rn-armory-spinnaker/armoryspinnaker_v2-22-0/#suffix-no-longer-added-to-jobs-created-by-kubernetes-run-job-stage) in Armory 2.22.
+
+To continue adding a random suffix to jobs created by the Kubernetes Run Job stage, use the `metadata.generateName` field in your Kubernetes job manifests. For more information, see [Generated values](https://kubernetes.io/docs/reference/using-api/api-concepts/#generated-values).
+
+{{< include "breaking-changes/bc-k8s-version-pre1-16.md" >}}
+
+{{< include "breaking-changes/bc-k8s-infra-buttons.md" >}}
+
+## Known issues
+<!-- Copy/paste known issues from the previous version if they're not fixed. Add new ones from OSS and Armory. If there aren't any issues, state that so readers don't think we forgot to fill out this section. -->
+
+{{< include "known-issues/ki-bake-var-file.md" >}}
+{{< include "known-issues/ki-lambda-ui-caching.md" >}}
+
+## Fixed issues
+
+* Fixed an issue where you could not edit AWS server groups with the **Edit** button in the UI. The edit window closed immediately after you opened it.
+* Fixed an issue where names for ECS server groups were not being recognized.
+* Fixed an issue where the first deployment of a manifest that used a blue/green strategy failed to find the service load balancer. This occurred if the service is in the same manifest as the workload getting deployed.
+* Fixed an issue where an error occured when deleting a CRD through the UI.
+* Fixed an issue where some caching agents for the SQL agent scheduler never executed. This could lead to the UI not reflecting changes caused by pipelines or pipelines not running if they were configured to use a Docker trigger. This issue affected all cloud providers and Docker triggers.
+* Fixed an issue where the **Expected Artifact** did not function as intended with
```

Comment:
Sentence doesn't seem to be complete here...is something missing?

Reply:
Rewrote based on the code PRs linked in https://armory.slab.com/posts/1-26-oss-service-summary-gfnv76fw#ext
  [ORCA] appEngine/docker: Fix appengine docker artifact mapping [a559994a](https://github.com/spinnaker/orca/commit/a559994a7dd51a5a84034df5e3c4a60b116fb9ba)